lib/cosmos/dart/script/dart_ingester.rb

```diff
@@ -0,0 +1,52 @@
+require File.expand_path('../../config/environment', __FILE__)
+require 'dart_packet_log_writer'
+require 'dart_logging'
+
+class DartInterfaceThread < Cosmos::InterfaceThread
+  attr_writer :packet_log_writer
+  attr_writer :log_type
+
+  def handle_packet(packet)
+    if packet.target_name == 'SYSTEM'.freeze and packet.packet_name == 'META'.freeze
+      Cosmos::Logger.info("#{@log_type}: #{packet.target_name} #{packet.packet_name}")
+      # Update Current Value Table Used By Packet Log Writer
+      cvt_packet = Cosmos::System.telemetry.update!(packet.target_name, packet.packet_name, packet.buffer)
```

Comment:
What is the point of updating the CVT right here? I see code in PacketLogWriter in start_new_file_hook which calls System.telemtry.packet('SYSTEM','META') but that's only if the first packet IS NOT a SYSTEM META packet which doesn't seem possible considering how everything else works.

Reply:
It is just more robust because PacketLogWriter does read the packet from System and it should be up to date.   May not be required buy also shouldn't be removed.